src/AllCapsules.jsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react";
+import getCapsules from "./helpers/api"
+import styles from "./AllCapsules.module.css"
+import rocketActive from "./assets/rocketActive.svg"
+import rocketUnknown from "./assets/rocketUnknown.svg"
+import rocketDeactivate from "./assets/rocketDeactivate.svg"
+
+export function AllCapsules() {
+  const [capsules, setCapsules] = useState([])
+  useEffect(() => {
+    getCapsules('https://api.spacexdata.com/v3/capsules')
+      .then((json) => setCapsules(json)
+      )
+  }, [])
+  console.log(capsules);
```

Comment:
```suggestion
  const [capsules, setCapsules] = useState([])
  
  useEffect(() => {
    getCapsules('https://api.spacexdata.com/v3/capsules')
      .then((json) => setCapsules(json)
      )
  }, [])
  console.log(capsules);
```
Esta chido pero esto pudo haber estado en un custom hook.